esmvaltool/diag_scripts/seaice_area_extents/seaicearea_trends.py

```diff
@@ -0,0 +1,172 @@
+"""Diagnostic script to plot minima and maxima trends.
+
+based on code from Anton Steketee's COSIMA cookbook notebook
+https://cosima-recipes.readthedocs.io/en/latest/DocumentedExamples
+    /SeaIce_Obs_Model_Compare.html
+"""
+
+import logging
+import os
+import matplotlib.pyplot as plt
+import numpy as np
+import pandas as pd
+import xarray as xr
+
+from esmvaltool.diag_scripts.shared import (run_diagnostic, save_figure,
+                                            save_data)
+from esmvaltool.diag_scripts.shared._base import get_plot_filename
+
+# This part sends debug statements to stdout
+logger = logging.getLogger(os.path.basename(__file__))
+
+
+def sea_ice_area(sic, area, coordls):
+    """Percent sic is in 0-100. Mulitply by portion so divide by 100."""
+    sic = sic / 100
+    # valid sic between 0.15 and 1
+    return (sic * area).where((sic >= 0.15) * (sic <= 1)).sum(coordls)
```

Comment:
I will leave this for now as moving forward we may rewrite these functions with a different package (iris) instead of xarray to be more aligned with esmvaltool and the functions included